### PR TITLE
Update docker base image, Django and Wagtail

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

**Review by simon-20 — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -27,15 +27,14 @@ RUN apk add postgresql-client && \
 
 RUN apk add python3-dev
 
-RUN apk add --no-cache python3 py3-pip && \
```
Comment (odscjames) [inline Dockerfile:30]: ??

File: Dockerfile [hunk 2]
```diff
@@ -27,15 +27,14 @@ RUN apk add postgresql-client && \
...
+RUN apk add --no-cache python3 py3-pip & \
  if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi && \
  if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3 /usr/bin/python; fi
 
 
-RUN apk add build-base libffi-dev libressl-dev && \
+RUN apk add build-base libffi-dev && \
     ln -sf /usr/bin/python3 /usr/bin/python && \
```
Comment (odscjames) [inline Dockerfile:36]: Missing a slash at the start of one of the ln's but also isn't these done above? (Tho the one above could use a full path just so it's clear and less error-prone.

I realise you didn't change this, I just noticed. I'll make an issue and you can do now or someone do later.

File: requirements.in [hunk 3]
```diff
@@ -8,19 +8,20 @@ django-import-export
 django-haystack
 django-modeltranslation
 django-prettyjson
-django-recaptcha3
+django-recaptcha3 @ git+https://github.com/bartsanchez/django-recaptcha3.git@313509e40163ce381e7aa3decd1de1ed9f71d380
```
Comment (odscjames) [inline requirements.in:11]: It would be great to have a comment saying why this is and what has to happen to go back to the main one

**Comment by odscjames** [comment]
I don't quite understand dev_public, it seems to be used for testing? Is it used for anything else?

File: Dockerfile (hunk 1 above)
Comment (Bjwebb) [reply Dockerfile:30]: Typo, I'll fix.

File: Dockerfile (hunk 2 above)
Comment (Bjwebb) [reply Dockerfile:36]: Good spot, I see there's an issue now at https://github.com/IATI/IATI-Standard-Website/issues/771

**Pull request merged**